lib/core/src/server/config/webpack.config.default.js

```diff
@@ -1,4 +1,45 @@
-import autoprefixer from 'autoprefixer';
```

Comment:
I think you meant to keep this

Reply:
No, I updated it to use `postcss-preset-env`.

https://github.com/storybooks/storybook/pull/4405/files/4a6fd1baf91c7f19324bc735c26aa981f4ac8410#diff-ba9b40ce49bd5df0209abd09fc032f58R28

Reply:
Very cool - haven't heart of this!

All of the below if for my incoming PR:

So if this is based upon `.browserlistrc`... What happens when their coverage disagrees with our hard-coded autoprefixer config?

Lastly, I couldn't find anything on supporting PostCSS within CRA 2.0

I'm certain the answer should be: "leave it to them to figure out in full control mode", but should we do anything further for the rare situation where CRA users eject and come up with their own PostCSS config? It should be simple enough, and it further justifies the need to modularize our increasingly complex style-related webpack config logic.

Reply:
I was just trying to stick to what the CRA config had. `postcss-preset-env` will look at the `browserlist` set in the `package.json`, so not sure where this differs from what was there before.

I agree the styles-related webpack config needs to be modularized. To be honest I picked up this PR as I thought it was going to be simple and would allow me to get my toes wet with the codebase. It's turning out to be a bit bigger of a change haha. The more I add the more I'm not sure if I'm conflicting with some of the other apps.

Wouldn't #4411 support PostCSS within CRA 2.0?

Reply:
Thanks, nitpicking is fine, I'm just trying to follow along lol.

It looks like the post-css is clashing with the `marko` package. Both mine and yours are failing that build. I'm not familiar with the `marko` framework so I'm not 100% sure but it looks like it's because we removed the `postcss: {}` line from the default config. https://github.com/storybooks/storybook/blob/1f127f0b1d0cdd71708ed4864bc79055b1796f4f/lib/core/src/server/config/webpack.config.default.js#L24

I added that back in and now the `marko` package seems to work.

Reply:
Nice usage of the new suggestion feature 😉 

I haven't dived in at all, but I'm certain we can provide an escape hatch for one consumer if we're providing a green path for another.

This could turn into a slippery slope, but we are trying to support one of the biggest consumer bases via this PR.

@ndelangen  - know any Marko experts?